doc/nrf/ug_tfm.rst

```diff
@@ -0,0 +1,62 @@
+.. _ug_tfm:
+
+Running applications with Trusted Firmware-M
+############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+On nRF5340 and nRF9160, you can use Trusted Firmware-M (TF-M) as an alternative to :ref:`secure_partition_manager` for running an application from the non-secure area of the memory.
+
```

Comment:
For later: should have some information on which solution you should choose in what scenario and why.